ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -609,6 +609,38 @@ CH_SS_DX3SCHWC_CWSISOVM_CWRISOVM_CHWSWISOVM_CHWDPM_CWDT_CHWDT_RCKTM:
   - CHWDT
   - RCKTM
 
+CH_SS_DX4SWC_RWTM_REFSM2X_REFPM2X_CPC:
+  guid: "6de8a9fd-87c5-4190-b4e2-402ede91900d"
+  description: "Four stage air cooled chiller with return water temperature monitoring, circulation pump monitoring and two circuit refrigerant monitoring."
+  is_canonical: true
+  implements:
+  - CH
+  - SS
+  - DX4SWC
+  - RWTM
+  - REFSM2X
+  - REFPM2X
+  - CPC
+
+CH_SS_DX2SWC_CHDXVSC2X_RWTM_MWDPC_RWISOVM_REFSM2X_REFPM2X_CPC_INVIPWM2X_INVOPWM2X:
+  guid: "cf101310-f825-4637-9948-f036ecbb4210"
+  description: "Two stage air cooled chiller with dual compressor variable speed control, return water isolation valve, minimum differential pressure control, 
```

Comment:
nit: please remove carriage return in description text block (description text should just be on a single line)

Reply:
fixed